xbmc/system.h

```diff
@@ -159,25 +73,6 @@
 #include "PlatformInclude.h"
 #endif
 
-#if defined(TARGET_ANDROID)
-#define HAS_ZEROCONF
-#endif
-
-// GLES2.0 detected. Dont use GL!
-#ifdef HAVE_LIBGLESV2
-#undef HAS_GL
-#define HAS_GLES 2
-#endif
-
-#ifdef HAVE_LIBGLESV3
-#undef HAS_GLES
-#define HAS_GLES 3
-#endif
-
-#ifdef HAS_DVD_DRIVE
-#define HAS_CDDA_RIPPER
-#endif
-
 #define SAFE_DELETE(p)       do { delete (p);     (p)=NULL; } while (0)
```

Comment:
These can be moved to another (possibly new) header

Reply:
For a future PR ;)